Code/Commando/datasafe.h

```diff
@@ -233,8 +233,8 @@
 #define safe_int int
 #define safe_unsigned_int unsigned int
 
-#define safe_long long
-#define safe_unsigned_long unsigned long
+#define safe_long int
```

Comment:
this might be out of scope for this PR, but should we remove these `#defines`?  they look pretty useless to me.  If they are needed, they should probably be typedefs.

Reply:
They are used to switch between using some convoluted surrograte template types that are to limit hacking, but the safe_long ones can probably be replaced with safe_int and you are right, they should probably be typedefs, not defines. I'm going to say out of scope for this PR though, it already touches enough stuff 😅